core/src/main/resources/io/questdb/site/conf/server.conf

```diff
@@ -511,6 +511,7 @@ query.timeout.sec=60
 
 #telemetry.enabled=true
 #telemetry.queue.capacity=512
+#telemetry.hide.tables=true
```

Comment:
nit: might be a good idea to add a brief description for this prop.

Reply:
added comments here: https://github.com/questdb/questdb/pull/1951/commits/055fe50d49603384f583b83dbcd506eda98d4940